test/hummingbot/connector/exchange/beaxy/test_beaxy_api_order_book_data_source.py

```diff
@@ -0,0 +1,93 @@
+import asyncio
+import json
+from typing import Awaitable
+
+from unittest import TestCase
+
+from aioresponses import aioresponses
+
+from hummingbot.connector.exchange.beaxy.beaxy_api_order_book_data_source import BeaxyAPIOrderBookDataSource
+from hummingbot.connector.exchange.beaxy.beaxy_constants import BeaxyConstants
+
+
+class BeaxyAPIOrderBookDataSourceTests(TestCase):
+
+    def tearDown(self) -> None:
+        BeaxyAPIOrderBookDataSource._trading_pair_symbol_map = {}
+        super().tearDown()
+
+    def async_run_with_timeout(self, coroutine: Awaitable, timeout: float = 1):
+        ret = asyncio.get_event_loop().run_until_complete(asyncio.wait_for(coroutine, timeout))
+        return ret
+
+    @aioresponses()
+    def test_trading_pairs_initialization(self, mock_api):
+        url = BeaxyConstants.PublicApi.SYMBOLS_URL
+        resp = [
+            {
+                "symbol": "BTCUSDT",
+                "name": "BTCUSDT",
+                "minimumQuantity": 0.01,
+                "maximumQuantity": 2509.41,
+                "quantityIncrement": 1.0E-7,
+                "quantityPrecision": 7,
+                "tickSize": 1.0E-7,
+                "baseCurrency": "BTC",
+                "termCurrency": "USDT",
+                "pricePrecision": 7,
+                "buyerTakerCommissionProgressive": 0.25,
+                "buyerMakerCommissionProgressive": 0.15,
+                "sellerTakerCommissionProgressive": 0.25,
+                "sellerMakerCommissionProgressive": 0.15,
+                "type": "crypto",
+                "suspendedForTrading": False},
+            {
+                "symbol": "ETHBTC",
```

Comment:
I propose we make one of the pairs something like `base = "A"; quote = "B"`, which will fail on the previous implementation, but will pass with this implementation and will demonstrate that any case can be handled now